package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@types/koa": "^2.0.48",
+    "@types/koa-bodyparser": "^4.3.0",
```

Comment:
All @types should be `devDependencies` instead, since they are not really needed at runtime.

Reply:
Done